shared-bindings/displayio/Display.h

```diff
@@ -45,33 +45,28 @@ void common_hal_displayio_display_construct(displayio_display_obj_t* self,
     uint8_t set_column_command, uint8_t set_row_command, uint8_t write_ram_command, uint8_t set_vertical_scroll,
     uint8_t* init_sequence, uint16_t init_sequence_len, const mcu_pin_obj_t* backlight_pin, uint16_t brightness_command,
     mp_float_t brightness, bool auto_brightness,
-    bool single_byte_bounds, bool data_as_commands);
+    bool single_byte_bounds, bool data_as_commands, bool auto_refresh, uint16_t native_frames_per_second);
 
-int32_t common_hal_displayio_display_wait_for_frame(displayio_display_obj_t* self);
+bool common_hal_displayio_display_show(displayio_display_obj_t* self,
+                                       displayio_group_t* root_group);
 
-bool common_hal_displayio_display_show(displayio_display_obj_t* self, displayio_group_t* root_group);
+// Times in ms.
+void common_hal_displayio_display_refresh(displayio_display_obj_t* self, uint32_t target_frame_time, uint32_t maximum_frame_time);
```

Comment:
How about renaming the parameters to ``target_frame_ms`` and ``maximum_frame_ms`` — then you don't need that comment.